cassandra: fix cqlsh

noplay opened the pull request:
Actually the cassandra driver used for cqlsh is not fully compatible
with current cassandra.

The result is when you try to user SELECT you will have an error:
'ResultSet' object has no attribute 'column_types

See this discussion on Stackoverflow about the issue:
http://stackoverflow.com/questions/42558686/resultset-object-has-no-attribute-column-types

This commit bump the python cassandra driver to the last release.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

neutric commented:
@noplay Thanks for your contribution! It be of great help if the resources `setuptools`, `Cython`, and `thrift` were also updated. Would you mind?

noplay commented:
I just updated cython, thrift and for setuptools I changed the version for the same version use in the Python formula.

More recent setuptools version seem to have trouble at installation.

fxcoudert commented:
I think this warrants a `revision` bump

noplay commented:
How to bump the revision?

I add revision 1 in the formula or does it need to be a different value?

Le mar. 7 mars 2017 à 09:57, FX Coudert <notifications@github.com> a écrit :

> I think this warrants a revision bump
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/10650#issuecomment-284660728>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVFXVxJz82cknRD4ldCvxilMSboG_3mks5rjRwHgaJpZM4MUjip>
> .
>

fxcoudert commented:
Yes, `revision 1`

noplay commented:
Done

Le mar. 7 mars 2017 à 10:33, FX Coudert <notifications@github.com> a écrit :

> Yes, revision 1
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/10650#issuecomment-284669177>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVFXdPJ3uU-WLkfbf_82eQmadCkIzGdks5rjSSCgaJpZM4MUjip>
> .
>

fxcoudert commented:
Thanks @noplay for your contribution!

SrihariThalla commented:
Thanks, @noplay for fixing the issue! I'm so sorry that I [overlooked](https://github.com/Homebrew/homebrew-core/pull/10434) the dependency version bumps.

@smec-cgint @bcalmac Hope the issue is cleared for you now 😅
